packages/fxa-settings/src/components/Settings/PageRecoveryKeyCreate/index.tsx

```diff
@@ -2,49 +2,112 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { RouteComponentProps, useNavigate } from '@reach/router';
-import { useFtlMsgResolver } from '../../../models';
-import { usePageViewEvent } from '../../../lib/metrics';
-import VerifiedSessionGuard from '../VerifiedSessionGuard';
 import { HomePath } from '../../../constants';
+import { usePageViewEvent } from '../../../lib/metrics';
+import { useAccount, useFtlMsgResolver } from '../../../models';
+import FlowRecoveryKeyConfirmPwd from '../FlowRecoveryKeyConfirmPwd';
+import FlowRecoveryKeyDownload from '../FlowRecoveryKeyDownload';
 import FlowRecoveryKeyInfo from '../FlowRecoveryKeyInfo';
+import VerifiedSessionGuard from '../VerifiedSessionGuard';
+
+const viewName = 'settings.account-recovery';
 
 export const PageRecoveryKeyCreate = (props: RouteComponentProps) => {
-  const navigate = useNavigate();
+  usePageViewEvent(viewName);
+
+  const account = useAccount();
   const ftlMsgResolver = useFtlMsgResolver();
+  const navigate = useNavigate();
+
+  const [currentStep, setCurrentStep] = useState<number>(1);
+  const [formattedRecoveryKey, setFormattedRecoveryKey] = useState<string>('');
+
   const goHome = () => navigate(HomePath + '#recovery-key', { replace: true });
-  const [currentStep, setCurrentStep] = useState(1);
-  usePageViewEvent('settings.account-recovery');
+  const numberOfSteps = 4;
+
   const localizedPageTitle = ftlMsgResolver.getMsg(
     'recovery-key-create-page-title',
     'Account Recovery Key'
   );
 
-  /*
-    The content here will obviously be replaced as we complete the separate views for this flow. This page will use the same pattern as the example wizard in storybook for `FlowContainer`. All steps will be separate components which use the `FlowContainer` and accept `currentStep` and `setCurrentStep` as the props necessary to move the user through the flow.
-  */
+  const localizedBackButtonTitle = ftlMsgResolver.getMsg(
+    'recovery-key-create-back-button-title',
+    'Back to settings'
+  );
+
+  const navigateBackward = () => {
+    navigate('/settings');
+  };
+
+  const navigateForward = (e?: React.MouseEvent<HTMLElement>) => {
+    e?.preventDefault();
+    if (currentStep + 1 <= numberOfSteps) {
+      setCurrentStep(currentStep + 1);
+    } else {
+      navigate('/settings');
+    }
+  };
+
+  // Redirects to settings if a recovery key is already set
+  // TODO change to redirect to 'PageRecoveryKeyChange' once created
+  useEffect(() => {
+    if (account.recoveryKey && !formattedRecoveryKey) {
+      navigate(HomePath, { replace: true });
+    }
+  }, [account, formattedRecoveryKey, navigate]);
+
+  // TODO prevent page refresh? currently refreshing breaks user from flow
```

Comment:
Oof yeah, if we want to persist the state on refresh and keep users on the same page, I think we might have to do something gross like stick it in local/session storage. Anything else I can think of off the top of my head is going to clear on refresh.

Reply:
I've added a comment to revisit this behaviour (and decide if it warrants a change) in FXA-7239